### PR TITLE
Update UG for AddCommands and SelectCommands

### PR DESCRIPTION
**Pull request by TZL0**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -30,11 +30,146 @@ It helps to keep track of patient data, patient appointments and patient bills f
...
+
+`Command word + <prefix><input> ...`
+
+#### Parameter List
+
+|     | Prefix  | Meaning       | Input Constraint                                                                                       |
+|-----|---------|---------------|--------------------------------------------------------------------------------------------------------|
+| `*` | `n/`    | Name          | 1. Non-empty alphanumeric characters and spaces<br/> 2. Must be different from existing patient's name |
+| `*` | `p/`    | Phone number  | Numbers with at least 3 digits                                                                         |
+| `*` | `e/`    | Email address | `local-part@domain`                                                                                    |
```
Comment (guokweijie) [inline docs/UserGuide.md:58]: i dont think local-part is understood by user

File: docs/UserGuide.md [hunk 2]
```diff
@@ -30,11 +30,146 @@ It helps to keep track of patient data, patient appointments and patient bills f
...
+
+`+`  Accept multiple
+
+#### Examples:
+
+* `ap n/Bernice Yu a/#11-330, blk 775, Bishan e/b.yu@nus.edu.sg p/80880011 t/NUS t/staff` adds such patient.
+
+<img src="images/addcommand/ap1.png" width="800px" height ="400px">
+
+* `ap n/Bernice Yu a/#01-01, blk 1, Changi Villege e/b.yu@ntu.edu.sg p/80880011 t/NTU t/staff` is unable
```
Comment (guokweijie) [inline docs/UserGuide.md:75]: don't know if its necessary to show this duplicate patient

**Review by guokweijie — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (TZL0) [reply docs/UserGuide.md:58]: Same one in MESSAGE_CONSTRAINT

File: docs/UserGuide.md (hunk 2 above)
Comment (TZL0) [reply docs/UserGuide.md:75]: To show the name must be unique

**Comment by TZL0** [comment]
See above

**Review by SpecOps2016 — APPROVED** [review]
LGTM

**Pull request merged**